docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
```

Comment:
The node docs talk about running npm install as one of the dockerfile steps, although it's not exactly the steps here either:

https://nodejs.org/en/docs/guides/nodejs-docker-webapp/

I think if we're looking for examples to follow we should see what other npm packages as docker images are doing.